lib/article_json/export/apple_news/elements/text.rb

```diff
@@ -14,6 +18,23 @@ class Text < Base
           def export
             super.to_s
           end
+
+          # @param [Nokogiri::XML::NodeSet] text
+          def create_text_nodes(text)
+            Nokogiri::HTML.fragment(sanitize_text(text).gsub(/\n/, '<br>')).children
+          end
+
+          # Removes UNSUPPORTED_TAGS from text
+          #
+          # @param [String] text
```

Comment:
According to the other methods, `text` is a `Nokogiri::XML::NodeSet`.
Shouldn't we be able to iterate over it directly, without creating another `Nokogiri::HTML.fragment`?

Reply:
OK, Apparently the method doc here is wrong: https://github.com/Devex/article_json/blob/master/lib/article_json/export/common/html/elements/text.rb#L46